demos/2d/space_shooter/level.tscn

```diff
@@ -29,6 +30,27 @@ tile_data = IntArray( 16, 0, 17, 0, 18, 0, 19, 0, 20, 0, 21, 0, 22, 0, 23, 0, 24
 
 [node name="rail" parent="." instance=ExtResource( 2 )]
 
+[node name="ship" parent="rail"]
```

Comment:
Did you make the changes below consciously, or did they happen automatically upon save?